### ISSUE TITLE
Fix accessibility of reorder view

### ISSUE THREAD

**Issue by sw23** [anwilli5/coin-collection-android-US#45]
We're currently using a view where users have to drag to re-order, however that mechanism doesn't work well for assistive technologies.  I think adding some up and down arrow buttons in addition to the dragging should help
